.github/workflows/deploy-ui.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: |
           cd ${{env.AZURE_WEBAPP_PACKAGE_PATH}}
           npm install
-          npm run build
+          CI='' npm run build
```

Comment:
What does this do?

Reply:
this fixes the bug- the action is failing because of "Treating warnings as errors because process.env.CI = true." and adding this fixes that.

Reply:
Thanks for the clarification! Yes, it does fix it!